docs/content/authors.md

```diff
@@ -314,7 +314,7 @@ put them back in again.` >}}
   * Barry Muldrey <barry@muldrey.net>
   * Sebastian Brandt <sebastian.brandt@friday.de>
   * Marco Molteni <marco.molteni@mailbox.org>
-  * Ankur Gupta <ankur0493@gmail.com> <7876747+ankur0493@users.noreply.github.com>
+  * Ankur Gupta <7876747+ankur0493@users.noreply.github.com>
```

Comment:
Note this at the top of authors.md

```
{{< rem `email addresses removed from here need to be addeed to
bin/.ignore-emails to make sure update-authors.py doesn't immediately
put them back in again.` >}}
```

So you need to put the emails you want ignored into `bin/.ignore-emails` as well.

Can you do the change to authors.md and .ignore-emails in a separate commit please? I'm happy to have them in this PR or a new one whichever is easiest for you.

Reply:
I removed the change from this PR - will create a separate one for the authors.md change.